Working Open statement

dcwalk opened the pull request:
This attempts to synthesize the notes in two places:

- loomio thread: https://loomio.cryptography.dog/d/2V7FvXbT/operational-transparencies-to-the-public
- feedback from 2019-04-17 all hands: https://github.com/hyphacoop/december-retreat/blob/master/2019-04-17-all-hands-meeting.md

patcon commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.
```
> (and exactly what)

Not sure how to read this. Do you mean the tools/spaces/docs we keep open? If so, might "(and where)" capture the sentiment more clearly?

uditvira replied:
```suggestion
Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.
```

patcon commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
+- Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.
```
<3 

Correction: *negotiating

uditvira replied:
```suggestion
- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.
```

patcon commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
+- Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.
+- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay our our practices of consent and privacy
```
*out our

dcwalk replied:
```suggestion
- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay out our practices of consent and privacy
```

patcon commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
+
+Legend:
+👀 - View only
```
Might it be more clear to annotate the "edit" access? I feel like the default assumption is that open is "viewable", so it's the "edit" escalation that's more clear to see noted :)

patcon replied:
Or we could just be explicit, and have a notation for both?

👀 view-only
📝 view & edit

uditvira replied:
> at any point
Does this mean "for any reason" as well?

dcwalk replied:
I like the explicit double notation @patcon! will add in

dcwalk replied:
> > at any point
> 
> Does this mean "for any reason" as well?

@uditvira -- I'm not sure I'm seeing the crux of the difference here. Could you say more? 

I thought this captured the conversation we had: currently working with this policy and there is no mechanism explicated for approving/rejecting a request so any request is honoured.

Just to restate my vibes from the convo: in time we should revisit and set norms of what stays open + a more explicit consent process (to steal @patcon's wording) for the things that we've noticed as the sites where we need differential shading. I think over time we'll get to a point where we recognize when that happens. As we learn I think we'll need to have some discussion and reflection about the reasons we are okay with around privacy.

uditvira replied:
> at any point

... in time?

I assumed you were referring to an occasion (before, during, or after), which does not describe the scope of and limits to redactions

> currently working with this policy and there is no mechanism explicated for approving/rejecting a request so any request is honoured

This is the clarification I was looking for, thanks. Yes, I agree there is no need to systematize more explicitly for now

dcwalk replied:
Okay. If additional words would make things more clear @uditvira please propose a friendly amendment!

patcon commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+
+- Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀
+- Aggregated financal statements 👀
+- Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀
```
Is this repetition of "aggregated financial statements" intended to capture the separate org-wide vs project-level scope? If so, perhaps we could make this more clear

dcwalk replied:
It is intended to speak to project finances, can you suggest a wording to improve this?

patcon replied:
- Scope: organization. Aggregate financial statements 👀
- Scope: project. Deliverables*, task tracking, meeting notes, and aggregated financial statements 👀

patcon commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+
+**Require consent before releasing**:
+
+- Audio, visual (photography) and video recordings
```
Curious if the essence of this is visual representation of someone... As in, is it mostly likely that someone would allow a recording, so long as their face isn't present? Happy to defer this, but would like to eventually discuss framing this to model how folks usually conceptualize progressive consent to transparency :)

- audio recording of spoken word
- image or video representation of person

dcwalk replied:
Can you link to a resource on this? Super curious if there is intentionally some implicit degree difference by placing these as separate?

I was not thinking the essence was visual representation, with tri-council research ethics and in my previous work, the emphasis has been on ability to re-identify. Audio can be more identifiable than an image (in my experience). In terms of progressive consent-- the bullets you suggest doesn't reflect my experience // research consent forms and convos I've been in.

Happy to chat more but I think this might be a little premature?

patcon replied:
Hm. So your experience is not that images are perceived as more sensitive than audio? I've witnessed people avoid being on camera, but will ask questions from the audience knowing their voice is livestreamed. I'm not sure whether it meaningfully makes a difference in ethics, as both are perhaps equally likely to re-identify, but I've understood visual image to be more closely guarded than audio.

Where the rubber hits the road on this is that I can imagine someone consenting to audio of meeting being shared, but not video. I'm interested in the perceived baby-steps that might allow people to feel secure and in-control with sharing. Also, re-identificiation can happen from a verbatim transcript via [stylometric analysis](https://en.wikipedia.org/wiki/Stylometry), but I definitely have an assumption that most people feel a great deal more comfortable sharing a written transcript.

Sorry, no resources to share here, just my own perceptions.  Though Audrey Tang does have a [transparency policy](https://talk.pdis.nat.gov.tw/t/principles-for-handling-official-visits-to-digital-minister-audrey-tang/70) that favours at least one artifact be made public by default -- written, audio, or video -- but allows that to be chosen by person, presumably based on level of comfort and preference.

dcwalk replied:
I'm speaking from a (narrow) research context, but yeah wrt to photos v. audio. Video is a much higher bar and folks perception of that has many different facets. 

I'm going to resolve this here because I'v linked from #5 -- we'll definitely need to pick this up later, maybe with a project on deck? to help navigate

uditvira commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
```
What does the phrase "new economy" refer to? It was historically used in relation to [internet related innovation](https://www.nber.org/papers/w7833), however, I think you might be referring to [something like this](https://neweconomy.net/about/what-is-the-new-economy)?

patcon replied:
Good point. I'd be satisfied with a link out to something, rather than diving into the meaning in this doc itself, if that works for others

patcon replied:
Or we could start a glossary in the handbook, and start corralling this stuff :)

dcwalk replied:
Yeah I was borrowing "new economy" as shorthand for the three aspects they mention: " just, sustainable, and democratic future" + solidarity economy etc... I think we can use a non-jargon word here (and maybe crib from our mission/vision), but think a glossary of things would be good? I'm filing an issue of glossary terms here --> #4

dcwalk replied:
Okay added suggested language below, I'm resolving this for now

patcon commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+
+**Default closed to working groups and project members**:
+
+- Full personal/financial information of members
```
Does this reflect a discussion about sharing salaries internally? As in, do we want to explicitly refer to salaries in this doc? (I missed one of these all-hands meetings)

dcwalk replied:
I don't think we have had a discussion about "sharing" salaries but we have had a discussion about setting salaries floors as well as a max pay differential ratio (e.g., folks paid minimum XX/hour; max difference 1:2), I don't think it would be unreasonable to be very explicit about salaries and our pay guidelines, but that is different than actually sharing someone's YTD earnings.

My intent with this line (and I think a reflection of discussion about personal and financial information) what that it was for things like: home addresses, bank accts, SIN, health stuff (say if we have benefits and there is some minimum coordination of knowing OHIP details, or say a leave, or terrible case someone gets injured on the job... not like actually anyones health data), or also potential "HR" info. 

Some (scant) links on this: 
https://www.priv.gc.ca/en/privacy-topics/privacy-at-work/
https://www.minkenemploymentlawyers.com/employment-law-issues/workplace-privacy-an-increasingly-important-issue-in-the-information-age/

patcon replied:
Ah cool! Thanks! Adding that "for example" bit would def clarify this

dcwalk replied:
added!

patcon commented:
Playing around with bringin a sketch I made into Google Draw, which can be embedded into a mardown doc: https://docs.google.com/drawings/d/1rcZfyC73KLux6zu0k6m_km_LLkhDyr94pbzbiBhjNis/edit?usp=sharing

This doesn't need to be in scope of this PR, but as I'm thinking through how to mentally model this in simplest terms, I'm wondering if the distinction is less edit/view/none, but rather participate/view/none. And really, ability to "participate" is perhaps more reflective of what we're getting at, whereas "edit" is really like an implementation term from tech that is object-centred, not experiential :) And then, I participate in a document, I participate in loomio decisions, I participate in chat, etc etc. (and I don't "edit" chat or email)

dcwalk commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
```
Proposed revision with language from our mission:
```suggestion
- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
```

dcwalk commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
+
+Legend:
+👀 - View only
```
```suggestion
👀  view only
📝  view and edit
```

dcwalk commented:
> A thought came to me while I was playing around with pporting a paper sketch I made into Google Draw, which can be embedded into a markdown doc:
> 
> [![Diagram](https://camo.githubusercontent.com/44c0a08bf4ed4ac725e9f1965dabce37c3fad88c/68747470733a2f2f646f63732e676f6f676c652e636f6d2f64726177696e67732f642f652f32504143582d31765247332d71756d4731475776312d4d66335262714c6258444452674358394f7035627836576a6262476b68396f5130345a4e66446a686e73647a7742647a333138384a6a6b535a344a70585a71322f7075623f773d363030)](https://docs.google.com/drawings/d/1rcZfyC73KLux6zu0k6m_km_LLkhDyr94pbzbiBhjNis/edit?usp=sharing)
> 
> This doesn't need to be in scope of this PR, but here's the thought, as I'm thinking through how to mentally model this in simplest terms: I'm wondering if **the distinction is less edit/view/none, but rather participate/view/none**. And really, ability to "participate" is perhaps more reflective of what we're getting at, whereas "edit" is really like an implementation term from tech that is object-centred, not experiential :) And then, I participate in a document, I participate in loomio decisions, I participate in chat, etc etc. (and I don't "edit" chat or email)

@patcon -- I'm happy to shift to using the wording "participate" over edit, you are right that seems to map better to the experience.

dcwalk commented:
Thanks for the feedback @uditvira and @patcon -- cleaned up the minor stuff and made some more suggestions for your response!

patcon commented:
Heh, I used this as an excuse to browse the emoji list. If we do want to use "participate", then these feel like they could be options for capturing that. ( 👀 `:eyes:` seems great for "view")
📝 `:memo:`
⚡️ `:zap:`
🎮 `:video_game:`
⚽️ `:soccer:`
🥏  a frisbee
🎲 `:game_die:`
🎟 `:tickets:`
🎤 `:microphone:`
🚀 `:rocket:`
🐝 `:bee:`

dcwalk commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+
+**Default closed to working groups and project members**:
+
+- Full personal/financial information of members
```
```suggestion
- Full personal/financial information of members (e.g., home addresses, banking information, and social insurance numbers)
```

dcwalk commented on working-open.md:
```diff
@@ -0,0 +1,46 @@
+
+**Require consent before releasing**:
+
+- Audio, visual (photography) and video recordings
```
```suggestion
- Audio, visual, and video recordings (e.g., photographs or recorded interviews)
```

dcwalk commented:
@patcon and @uditvira can I get a (hopefully!) final review and sign-off from you? I addressed all remaining feedback--

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+
+## Why?
+
+- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
```
```suggestion
- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods
```

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+## Why?
+
+- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
+- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.
```
```suggestion
- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions
```

dcwalk replied:
You'd like to remove the periods?

dcwalk replied:
Reviewed and we err on the side of periods at the end of full sentences, including bullet points, both in this doc and in the other committed material. I'd rather add the one missing period than remove all of these

patcon replied:
I ever-so-slightly prefer punctuation on bullets too :) Esp with markdown. If you agree, perhaps mark these as resolved @benhylau?

benhylau replied:
@dcwalk I have no preference other than being consistent.

dcwalk replied:
Okay, resolving these suggestions then

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+
+# How and what?
+
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
```
```suggestion
- We currently work with an [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
```

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+- Scope: Project
+  - Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀
+
+* *This would include source repositories and code (e.g., on GitHub) and other process materials based on consensus on licensing and usage constraints.*
```
```suggestion
* *This would include source repositories and code (e.g. on GitHub) and other process materials based on consensus on licensing and usage constraints.*
```

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+  - Finances (budget, expenses, and cash flow projections) and audit records 👀
+  - Potential opportunities and leads 📝 
+- Scope: Project
+  - Project proposal(s), statement(s) of work (in-progress/signed), in-progres deliverables, and finances (budget, expenses, and cash flow projections) 👀
```
```suggestion
  - Project proposal(s), statement(s) of work (in-progress/signed), in-progress deliverables, and finances (budget, expenses, and cash flow projections) 👀
```

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+**Default closed to specific working groups and project members**:
+
+- Scope: Organization
+  - Full personal/financial information of members (e.g., home addresses, banking information, and social insurance numbers)
```
```suggestion
  - Full personal/financial information of members (e.g. home addresses, banking information, and social insurance numbers)
```

dcwalk replied:
I favour commas after e.g./i.e., it isn't a typo but a rather a distinct usage, see discussion: https://www.dailywritingtips.com/comma-after-i-e-and-e-g/ (I'd also say it fits in this text)

benhylau replied:
Not saying this one is a typo, but I think without the comma is more common and so more likely to be consistently used. Same with 1 or 2 spaces after periods, etc. If you prefer having the comma we can aim for that in all our documents, but I definitely feel not having it is likely to stay consistent.

patcon replied:
We could use [`vale`](https://errata-ai.github.io/vale/) and [dangerCI gem](https://danger.systems/guides/what_does_danger_do.html) to encode our rules and do some of these evolving checks. Demo: https://github.com/patcon/dangerfile-test/pull/1

Style rule: https://github.com/patcon/dangerfile-test/blob/master/ci/vale/styles/HyphaCoop/Eg.yml

patcon replied:
I'm pretty indifferent about which, though I'll need to get used to using commas. But I'm fine with whatever conventions as long as we're on same page :)

(hence my appreciating the bot auto-correcting)

dcwalk replied:
Commas after "For example" (e.g.) and "Further" (i.e.) tends to be grammatically correct when used without abbreviations. In my experience, but this is based on a lot of academic reading I recognize, commas are the more frequent usage.

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+- Scope: Organization
+  - Full personal/financial information of members (e.g., home addresses, banking information, and social insurance numbers)
+  - Personal/financial information involving external parties 📝 
+  - Administration privileges including financal for services and infrastructure 📝 
```
```suggestion
  - Administration privileges including financial for services and infrastructure 📝 
```

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+
+**Require consent before releasing**:
+
+- Audio, visual, and video recordings (e.g., photographs or recorded interviews)
```
```suggestion
- Audio, visual, and video recordings (e.g. photographs or recorded interviews)
```

benhylau commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+
+- Scope: Organization
+  - Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀
+  - Aggregated financal statements 👀
```
```suggestion
  - Aggregated financial statements 👀
```

benhylau commented:
Good to merge after typo changes are accepted

patcon commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+
+# How and what?
+
+- We currently work with an [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
```
Re-reading. Perhaps we could strike the use of "in camera"? It's a historic but perpetually-confusing term that means the opposite of what contemporary people assume. Here's an attempt to trim/simplify.

```suggestion
- For meeting notes, we accommodate redactions and going off-the-record. Participants can request to stop taking notes, or redact and correct notes after they have been taken.
```

patcon replied:
Members => participants, because this isn't just for members, right?

dcwalk replied:
I'm not sure about other folks, but if anyone has attended a more formally chaired meeting than an in camera process is not likely unfamiliar. I'm fine with clarifying language, I guess I wonder if we didn't already address your concerns with the following statements?

I'd also prefer we keep the temporal aspects: that this is our current practice.

patcon commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+
+## Why?
+
+- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
```
This line is getting at something like "other people can read and learn from our stuff", right? Having feels about [thing-explainer'ing](https://xkcd.com/thing-explainer/) some of this language, but don't want to be pedantic! :)

Wondering if conceiving an "audience" or some personas for this handbook would be helpful. Our parents? Children? A baby boomer investor? Tradespersons?

dcwalk replied:
This is our mission/vision statement.

dcwalk replied:
I'm going to leave this for now because we are our initial audience and this is our own mission statement! I've added your thoughts to #8

patcon commented on working-open.md:
```diff
@@ -0,0 +1,53 @@
+👀  view only  
+📝  view and participate  
+
+**Default open to all**:
```
This might be more explicit:

```suggestion
**Default open to public**:
```

dcwalk commented:
Okay, I've moved the discussion items into issues (referencing this PR) and added the period. I'm going to do ahead an merge this as the minor remaining suggestions can be substantively discussed in those issues and then applied to all handbook elements.

We're gonna merge this!
